jira/table_jira_issue.go

```diff
@@ -516,6 +516,28 @@ func searchWithContext(ctx context.Context, d *plugin.QueryData, jql string, opt
 	}
 	uv.Add("expand", options.Expand)
 
+	// Specify fields to prevent getting more data than necessary
```

Comment:
We should automatically detect these list from the Transform attribute. 
LGTM for now to build the image.